xcodeproj/internal/linker_input_files.bzl

```diff
@@ -366,6 +366,13 @@ def _process_linkopt(linkopt, *, triple):
         if opt.endswith(".swiftmodule"):
             opt = opt + "/{}.swiftmodule".format(triple)
 
+        # Process paths in the --flag=value format
+        flag, sep, value = opt.partition("=")
+        if value:
```

Comment:
I think you can move this up and only do the path prepend logic once?

Reply:
Not sure if how I changed it is like you meant it, but I couldn't find a way to make it more readable. Let me know how it looks!